include/json/writer.h

```diff
@@ -187,6 +190,8 @@ class JSONCPP_DEPRECATED("Use StreamWriterBuilder instead") JSON_API FastWriter
   bool yamlCompatibilityEnabled_;
   bool dropNullPlaceholders_;
   bool omitEndingLineFeed_;
+  UInt precision_;
```

Comment:
We cannot change the size of a class in a public header.

Let's concentrate on the newer CharReader and StreamWriter versions. (Everybody hates them because users do not like to type a few extra characters, but those classes be altered as often as we like without impacting binary-compatibility.)

Reply:
Fixed.